app/pages/search.py

```diff
@@ -58,20 +60,19 @@ def search_index_page():
             .join(StudyUser, isouter=True)
             .where(publish_clause)
             .group_by(Study)
-            .order_by(Study.updatedAt.desc())
-            .limit(5)
+            .limit(_PER_PAGE)
         ).all()
 
     if studyIds:
         results = g.db_session.scalars(
             sql.select(Study)
             .where(Study.publicId.in_(studyIds))
-            .order_by(Study.updatedAt.desc())
+            .order_by(Study.createdAt.desc())
```

Comment:
Studies get updated in minor ways all the time and this shifts the ordering. Ordering by creation time seems a bit more legible (they're basically ordered from 6 to 1)